src/PowerStatus.h

```diff
@@ -59,9 +59,18 @@ class PowerStatus : public Status
     int getBatteryVoltageMv() const { return batteryVoltageMv; }
 
     /**
-     * Note: 0% battery means 'unknown/this board doesn't have a battery installed'
+     * Note: for boards with battery pin, 0% battery means 'unknown/this board doesn't have a battery installed'
      */
+#ifdef BATTERY_PIN
```

Comment:
I think we also need to check for HAS_PMU, otherwise T-Beams which do not have a battery pin but use the AXP chip would not report correctly

Reply:
I almost broke something :) Thanks for the reminder